app/models/user.rb

```diff
@@ -440,6 +422,24 @@ def self.try_to_auto_create_user(login, password)
     end
   end
 
+  private
```

Comment:
Wow, github's diff was kind of dumb here? I guess you just moved the `self` methods  but it identified the rest.